contacts/models.py

```diff
@@ -19,3 +20,25 @@ class Contact(models.Model):
     city = models.CharField(max_length=255, null=True, blank=True)
     state = USStateField(null=True, blank=True)
     zip_code = USZipCodeField(null=True, blank=True)
+
+class Note(models.Model):
+    note = models.TextField(blank=True, null=True)
+    created_on = models.DateTimeField(auto_now_add=True)
+    contact = models.ForeignKey(Contact, on_delete=models.CASCADE,)
+    phone_regex = RegexValidator(
```

Comment:
Because you have the foreign key relationship, you don't need to re-state the attributes of the Contact model. By pointing to the contact, you get access to all its attributes. So, for example, `note.contact.email`  would give you the email for this note's contact.